### PR TITLE
patches: update releases for October cycle

### PR DESCRIPTION
**Pull request by cpanato**
Update Patch releases for October cycle

/assign @justaugustus @puerco @Verolop @xmudrii 
/cc @kubernetes/release-managers

### CODE REVIEW

**Comment by cpanato** [comment]
should we update the EOL https://github.com/kubernetes/website/pull/29744/files#diff-3fbc599bc532616df1b24035ffaafddf482f185fdd2b04c3f1597e34d9ef62f3R138 to be the day of this Oct patch release? or we might make a final release in November?

File: data/releases/schedule.yaml [hunk 1]
```diff
@@ -67,11 +76,14 @@ schedules:
       targetDate: 2020-12-18
       note: "Tagging Issue https://groups.google.com/g/kubernetes-dev/c/dNH2yknlCBA"
 - release: 1.19
-  next: 1.19.15
-  cherryPickDeadline: 2021-09-10
-  targetDate: 2021-09-15
+  next: 1.19.16
+  cherryPickDeadline: 2021-10-15
+  targetDate: 2021-10-20
   endOfLifeDate: 2021-10-28
```
Comment (cpanato) [inline data/releases/schedule.yaml:82]: should we update the EOL to be the day of this Oct patch release? or we might make a final release in November?

**Comment by reylejano** [comment]
Preview of updates page:
- https://deploy-preview-29744--kubernetes-io-main-staging.netlify.app/releases/patch-releases/

File: data/releases/schedule.yaml (hunk 1 above)
Comment (xmudrii) [reply data/releases/schedule.yaml:82]: I'd agree with @puerco. Unless we have something critical, we should adhere to the defined guidelines and not release another patch release. In that case, I don't think it makes sense to add it to the schedule.

Comment (cpanato) [reply data/releases/schedule.yaml:82]: agree as well, will update the PR, thanks for the feedback!

**Review by xmudrii — APPROVED** [review]
/lgtm
/hold
for other folks to take a look

**Comment by cpanato** [comment]
if we need to change we can do in a follow up, remove the hold

/unhold

**Pull request merged**